test/release/examples/users-guide/base/configs2.compopts-fake

```diff
@@ -0,0 +1,8 @@
+chpl -sverbose=true configs2.chpl
+chpl -sverbose configs2.chpl
+chpl -sbitMask=0b00001111 -ssize=16 -sage=uint(16) configs2.chpl
+chpl -sage=real(32) configs2.chpl
```

Comment:
Great point.  It hadn't occurred to me that the use of the .execopts file was protecting me from issues like this...